explorer/src/components/account/TransactionHistoryCard.tsx

```diff
@@ -75,6 +77,12 @@ export function TransactionHistoryCard({ pubkey }: { pubkey: PublicKey }) {
             <Slot slot={slot} link />
           </td>
 
+          {hasTimestamps && (
+            <td className="text-muted">
+              {blockTime ? displayTimestampUtc(blockTime * 1000) : "---"}
```

Comment:
![Screen Shot 2021-01-26 at 5 43 58 PM](https://user-images.githubusercontent.com/1076145/105828419-12ff8680-5ffe-11eb-8f09-505d122bc56e.png)

It's not clear from the UI that this is a UTC timestamp

Reply:
Agreed -- this actually stems from "unlock day" where we decided to remove the mention of UTC when displaying cluster times. @mvines this should be safe/okay to add back in? Side effect: UTC would also be displayed on the cluster stats info on the front page.

Reply:
Yeah I'd prefer an explicit timezone (UTC wfm) everywhere now that the cluster time drift that was causing confusion in the past is resolved.

Reply:
On the transaction details page, we're displaying these as local time. I will follow suit here!

Reply:
@mvines sounds good. I'll update that in this PR.